### PR TITLE
adds wnFess's custom item

### PR DESCRIPTION
**Pull request by Casper3667**
https://baystation12.net/forums/threads/excavation-dufflebag-brendan-moon-wnfess.3465/#post-52591
should be it unless I forgot something.

### CODE REVIEW

File: config/custom_items.txt [hunk 1]
```diff
@@ -583,3 +583,11 @@ item_name: Clipboard folder
 item_icon: scottsfolder
 item_desc: A neat cross between a folder and a clipboard. It's made out of hard, very lightly textured plastic. There are little colored tabs jutting out, and various personal files inside. On the spine label, written neatly in red ink, is the owner's name - Emilia Scotts.
 }
+{
+ckey: wnfess
+character_name: Brendan Moon
+item_path: /obj/item/weapon/storage/backpack/dufflebag
+item_name: Excavation Dufflebag
+item_icon: duffle_exc
+item_desc: Yellow duffelbag with several black stripes on it as well as a warning sign."B.Moon" is sewn into the fabric just below the sign.
```
Comment (mwerezak) [inline config/custom_items.txt]: Probably should be a space after the first period

Comment (Casper3667) [reply config/custom_items.txt]: heh, ooops. I just copied directly from the thread.

**Comment by Raptor1628** [comment]
Hold on this.

**Comment by Raptor1628** [comment]
Never mind, this is good to go.

**Pull request merged**